tests/tests/test_basics.py

```diff
@@ -761,7 +765,7 @@ def check(self, cls, args, expected_status):
         assert status == expected_status
 
     def test(self):
-        self.check(self.Mysuite, [], 0)
+        self.check(self.Mysuite, [], 1)
```

Comment:
Please add a testsuite run with `--failure-exit-code=0`, as this case is now missing